cscs-checks/apps/cp2k/cp2k_check.py

```diff
@@ -132,46 +167,34 @@ def set_cpu_binding(self):
         self.job.launcher.options = ['--cpu-bind=cores']
 
 
-@rfm.parameterized_test(*([s, v]
-                          for s in ['small', 'large']
-                          for v in ['maint', 'prod']))
+@rfm.simple_test
 class Cp2kGpuCheck(Cp2kCheck):
-    def __init__(self, scale, variant):
-        super().__init__()
-        self.descr = 'CP2K GPU check (version: %s, %s)' % (scale, variant)
-        self.valid_systems = ['daint:gpu']
-        self.num_gpus_per_node = 1
-        if scale == 'small':
+    valid_systems = ['daint:gpu']
```

Comment:
I think that GPU/CPU "concept" tests should be part of the library and the tests should inherit from the appropriate base classes.